pkg/backends/jmeter/resources_test.go

```diff
@@ -93,19 +93,20 @@ func TestGetNamespaceFromInvalidName(t *testing.T) {
 }
 
 func TestPodResourceConfiguration(t *testing.T) {
-	c := &JMeter{
-		loadTest: &loadtestv1.LoadTest{
-			Spec: loadtestv1.LoadTestSpec{
-				MasterConfig: loadtestv1.ImageDetails{
-					Image: defaultMasterImageName,
-					Tag:   defaultMasterImageTag,
-				},
-				WorkerConfig: loadtestv1.ImageDetails{
-					Image: defaultWorkerImageName,
-					Tag:   defaultWorkerImageTag,
-				},
+	lt := loadtestv1.LoadTest{
+		Spec: loadtestv1.LoadTestSpec{
+			MasterConfig: loadtestv1.ImageDetails{
+				Image: defaultMasterImageName,
+				Tag:   defaultMasterImageTag,
+			},
+			WorkerConfig: loadtestv1.ImageDetails{
+				Image: defaultWorkerImageName,
+				Tag:   defaultWorkerImageTag,
 			},
 		},
+	}
+
+	c := &Backend{
```

Comment:
@boekkooi-fresh note that here I had to change from `JMeter` to `Backend` instead, to actually test those implementations.